### PR TITLE
Fix sorting order for find ER and hyperparameters (keyvalue fields)

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java [hunk 1]
```diff
@@ -27,23 +30,50 @@
...
-  private InternalFuture<QueryFilterContext> processKeyValueSort(
-      String key, boolean ascending, String fieldType) {
+  private QueryFilterContext processKeyValueSort(String key, boolean ascending, String fieldType) {
     var sql =
-        "select experiment_run_id as id, kv_value as value from keyvalue where entity_name=\"ExperimentRunEntity\" and field_type=:sort_field_type and kv_key=:sort_key";
+        "select experiment_run_id as id, kv_value as value from keyvalue where entity_name=:entityName and field_type=:sort_field_type and kv_key=:sort_key";
     var queryContext =
         new QueryFilterContext()
             .addBind(q -> q.bind("sort_field_type", fieldType))
+            .addBind(q -> q.bind("entityName", "ExperimentRunEntity"))
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java:56]:
```suggestion
            .addBind(q -> q.bind("entityName", "\"ExperimentRunEntity\""))
```

File: backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java [hunk 2]
```diff
@@ -27,23 +30,50 @@
...
+            sql, ascending, Collections.singletonList(new OrderColumn("value", ascending))));
+    return queryContext;
+  }
+
+  private QueryFilterContext processConfigHyperparametersSort(String key, boolean ascending) {
+    var hyperparameterFromBlobMappingSql =
+        "select distinct experiment_run_id as id, int_value, float_value, string_value from hyperparameter_element_mapping where entity_type=:entity_type and experiment_run_id IS NOT NULL and name=:name";
+    var queryContext =
+        new QueryFilterContext()
+            .addBind(q -> q.bind("entity_type", "ExperimentRunEntity"))
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java:69]:
```suggestion
            .addBind(q -> q.bind("entity_type", "\"ExperimentRunEntity\""))
```

**Review by conradoverta — APPROVED** [review]

File: backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java:56]: No need to wrap with `""` we are removing quotes as per @coryverta PR.

Comment (conradoverta) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java:56]: Are you absolutely sure? I believe his PR added the quotes

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java:56]: @conradoverta 's reference PR: https://github.com/VertaAI/modeldb/pull/2294/files

**Comment by coryverta** [comment]
I ultimately ended up removing the quotes after testing, they were only required when inside the main query string.  As a bound parameter they are unneccessary and end up getting stored as part of the value.

**Pull request merged**